src/interfaces/guild.interface.ts

```diff
@@ -9,4 +9,14 @@ export interface IGuildUpdateBody {
     ],
     period?: Date,
     isDisconnected?: boolean,
+}
+
+
+export interface IPermissionOverwrite {
+    id: string,
+    type: 'role' | 'member';
+    allow: string,
+    deny: string,
+    allow_new: string,
+    deny_new: string
```

Comment:
What is IPermissionOverwrite for?

Reply:
It s use for computing permission.
It will use in future works

Reply:
Okay